ppr-ui/src/store/store.ts

```diff
@@ -626,7 +626,9 @@ export const useStore = defineStore('assetsStore', () => {
     // Current state is to verify the property exists. Future state may be more granular dependent on type.
     return !!state.value.mhrInformation.lienRegistrationType
   })
-  const getMhrUnitNotes: ComputedRef<Array<UnitNoteIF>> = computed<Array<UnitNoteIF>>(() => {
+  const getMhrUnitNotes: ComputedRef<Array<UnitNoteIF | CancelUnitNoteIF>> = computed<
+    Array<UnitNoteIF | CancelUnitNoteIF>
+  >(() => {
     return state.value.mhrUnitNotes
   })
```

Comment:
formatting is a little interesting here, hard to decipher the computed type from broken up like that.

Reply:
Haha yea didn't even notice, I stopped using a formatter a long time ago, and only use to format long lines, I'll try a different formatter.